src/main/java/net/atos/zac/app/klanten/converter/RESTBedrijfConverter.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.app.klanten.converter;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import net.atos.client.kvk.model.KVKZoekenParameters;
+import net.atos.client.kvk.zoeken.model.Resultaat;
+import net.atos.client.kvk.zoeken.model.ResultaatItem;
+import net.atos.zac.app.klanten.model.bedrijven.RESTBedrijf;
+import net.atos.zac.app.klanten.model.bedrijven.RESTListBedrijvenParameters;
+
+public class RESTBedrijfConverter {
+
+
+    public KVKZoekenParameters convert(final RESTListBedrijvenParameters restListParameters) {
+        if (!restListParameters.isValid()) {
```

Comment:
Deze valiodatie is niet nodig, Als de RESTListBedrijvenParameters niet valid zijn dan zijn de KVKZoekenParameters dat ook niet en laat de KVK API dan maar de validatie doen. Wellicht dat die validatie nog eens veranderd in de toekomst? 
Deze validatie moet natuurlijk wel aan de voorkant in het invulscherm zitten

Reply:
neem ik mee in vervog ticket